### PR TITLE
Issue template for bugs

### PR DESCRIPTION
**Pull request by YTG1234**
For some reason GitHub closed #4.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+1. Scroll down to '....'
+1. See error
+
+## Expected behavior
+A clear and concise description of what you expected to happen (e.g. description of what happens in the Forge version if the bug is only here).
+
+## Screenshots
+If applicable, add screenshots to help explain your problem.
+
+## System information (please complete the following information):
```
Comment (PepperCode1) [inline .github/ISSUE_TEMPLATE/bug-report.md:31]: I think non-JiJed dependency versions and a list of other mods should be provided as well. Someone could report that connected textures don't work with Sodium, but that is not Create's bug.

**Review by PepperCode1 — CHANGES_REQUESTED** [review]

**Review by PepperCode1 — APPROVED** [review]

**Comment by TropheusJ** [comment]
add a section under System Information for an alt + shift + f3 debug screenshot. otherwise looks good to me.

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Screenshots
+Attach a screenshot of the ⌥+⇧+F3 screen to help us see your performance stats.
+
+If applicable, add additional screenshots to help explain your problem.
+
+## System information (please complete the following information):
+ - Fabric Loader version [e.g. `0.11.3`]
+ - Fabric API version [e.g. `0.32.0+1.16`]
+ - Cloth Config 2 version [e.g. `4.11.14`]
```
Comment (PepperCode1) [inline .github/ISSUE_TEMPLATE/bug-report.md:35]:
```suggestion
- LibBlockAttributes version [eg. `0.8.4`]
 - Cloth Config 2 version [e.g. `4.11.14`]
```

File: .github/ISSUE_TEMPLATE/bug-report.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+ - Commit hash [e.g. `c8b7bd4d7a065f29b02cb06b9a68f5a841c664a9`]
+
```
Comment (PepperCode1) [inline .github/ISSUE_TEMPLATE/bug-report.md:41]:
```suggestion
- List of all other installed mods, including versions

```

**Review by PepperCode1 — CHANGES_REQUESTED** [review]
Added LBA version and list of other mods. After this change this PR should probably be merged.

File: .github/ISSUE_TEMPLATE/bug-report.md (hunk 2 above)
Comment (YTG1234) [reply .github/ISSUE_TEMPLATE/bug-report.md:35]: oh wait you missed a space there

**Review by PepperCode1 — APPROVED** [review]

**Pull request merged**